scripts/qiita_test_install

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from os import environ
+from os.path import join, dirname, abspath
+from sys import platform, version as python_version, exit, executable, stdout
+from unittest import TestLoader, TextTestRunner, TestCase
+from smtplib import SMTP, SMTP_SSL
+
+core_dependency_missing_msg = (
+    '"%s" is missing and is a core requirement, for more information see the '
+    'Qiita Installation Guide: '
+    'https://github.com/biocore/qiita/blob/master/INSTALL.md')
+
+extra_info = (core_dependency_missing_msg + '. It is also possible that you '
+              'have an old version of this package, if so, please update to '
+              'the latest.')
+
+dependency_missing_msg = (
+    '"%s" is missing but this is _not_ a core requirement, for more '
+    'information see the Qiita Installation Guide: '
+    'https://github.com/biocore/qiita/blob/master/INSTALL.md')
+
+missing_deps_errors = []
+missing_deps_warnings = []
+
+try:
```

Comment:
That would have also worked, though note that this PR is now merged, so
we may want to open a separate issue about this.

On (Feb-03-15|13:06), Daniel McDonald wrote:
> > +    'Qiita Installation Guide: '
> > +    'https://github.com/biocore/qiita/blob/master/INSTALL.md')
> > +
> > +extra_info = (core_dependency_missing_msg + '. It is also possible that you '
> > +              'have an old version of this package, if so, please update to '
> > +              'the latest.')
> > +
> > +dependency_missing_msg = (
> > +    '"%s" is missing but this is _not_ a core requirement, for more '
> > +    'information see the Qiita Installation Guide: '
> > +    'https://github.com/biocore/qiita/blob/master/INSTALL.md')
> > +
> > +missing_deps_errors = []
> > +missing_deps_warnings = []
> > +
> > +try:
> 
> what about using `importlib`?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/805/files#r24039719